eland/tests/dataframe/test_metrics_pytest.py

```diff
@@ -424,3 +424,18 @@ def test_aggs_count(self):
         ed_count = ed_flights.agg(["count"])
 
         assert_frame_equal(pd_count, ed_count)
+
+    @pytest.mark.parametrize("numeric_only", [True, False])
```

Comment:
Can we add a test case that shows that `es_size` works as expected? (ie have a situation where there are more than 1 mode, set es_size>N and see that the dataframe is ==N, then set es_size<N and see that dataframe is es_size still.

In this same test case you can likely test the "fill" mechanic too

Reply:
Absolutely